src/sentry/replays/usecases/query/conditions/base.py

```diff
@@ -0,0 +1,57 @@
+from __future__ import annotations
```

Comment:
Literally this whole module exists to drop `expression` as the first argument of the method.